x-pack/solutions/security/plugins/security_solution/public/common/components/links/index.tsx

```diff
@@ -298,58 +298,81 @@ export interface NetworkDetailsLinkProps {
   ip: string | string[];
   flowTarget?: FlowTarget | FlowTargetSourceDest;
   isButton?: boolean;
-  onClick?: (e: SyntheticEvent) => void | undefined;
+  onClick?: (ip: string) => void;
   title?: string;
 }
 
-const NetworkDetailsLinkComponent: React.FC<NetworkDetailsLinkProps> = ({
-  Component,
-  children,
-  ip,
-  flowTarget = FlowTarget.source,
+const NetworkDetailsLinkComponent: React.FC<NetworkDetailsLinkProps> = ({ ip, ...restProps }) => {
+  // We see that sometimes the `ip` is passed as a string value of "IP1,IP2".
```

Comment:
can the same happen for arrays, i.e. `["IP1,IP2", "IP3,IP4"]`?

Reply:
I'm not sure to be honest. It's not something that I've seen before though.

Reply:
I'd say, let's not make the code more complex. I don't think an array of string with multiple values like that is a situation that we need to handle...